src/ivc/incrementally_verifiable_computation.rs

```diff
@@ -228,17 +231,89 @@ where
 
     pub fn verify<const T: usize, RP1, RP2>(
         &mut self,
-        _pp: &PublicParams<'_, A1, A2, T, C1, C2, SC1, SC2, RP1, RP2>,
-        _steps_count: usize,
-        _z0_primary: [C1::Scalar; A1],
-        _z0_secondary: [C2::Scalar; A2],
+        pp: &PublicParams<'_, A1, A2, T, C1, C2, SC1, SC2, RP1, RP2>,
+        num_steps: usize,
```

Comment:
Can it verify zero step? If not this should be `NonZeroUsize`

Reply:
zero step is the initial condition, there is no need to verify, we can make it NonZeroUsize.